src/openalea/strawberry/application/app_visualization.py

```diff
@@ -66,7 +82,18 @@ def on_change_tab_3d(widget, event, data):
     with plot3d_growth_developement:
         plot3d_growth_developement.clear_output()
 
+def on_change_geno_2d(widget, event, data):
+    id_selection_2d.items = [*range(len(get_vid_of_genotype(misc.all_mtg, [data]))) ]
```

Comment:
rather than [*range(...)] I will do list(range).
This is more explicit, isn't it?